playground/api/v1/api.proto

```diff
@@ -135,6 +135,7 @@ message PrecompiledObject{
   string name = 2;
   string description = 3;
   PrecompiledObjectType type = 4;
+  string pipeline_options = 5;
```

Comment:
```suggestion
  map<map<string, string>> pipeline_options = 5;
```

Reply:
Agree, this should be a map with string keys and values. The parsing of the string will be performed on the frontend

Reply:
As I understand we send a `string` to the frontend

Reply:
Why `map` into `map`?

Reply:
?? why map?

Reply:
Discussed with Aydar, we decided to have a string as it is now in this PR